core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -587,25 +587,12 @@ public Address getAddress() {
          log.tracef("dests=%s, command=%s, mode=%s, timeout=%s", recipients, rpcCommand, mode, timeout);
       Address self = getAddress();
       boolean ignoreLeavers = mode == ResponseMode.SYNCHRONOUS_IGNORE_LEAVERS || mode == ResponseMode.WAIT_FOR_VALID_RESPONSE;
-      List<Address> members = getMembers();
-      if (mode.isSynchronous() && recipients != null && !members.containsAll(recipients)) {
-         if (!ignoreLeavers) { // SYNCHRONOUS
-            Address suspect = recipients.stream().filter(a -> !members.contains(a)).findFirst().orElse(null);
-            CompletableFuture<Map<Address, Response>> future = new CompletableFuture<>();
-            future.completeExceptionally(new SuspectException(
```

Comment:
No, `JGroupsTransport.checkRsp()` still throws a `SuspectException` -- I had forgotten that `Responses` returns the entire `Rsp` and the `Response` is extracted later.